src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -142,15 +167,20 @@
             "name": "id",
             "in": "path",
             "required": true,
-            "style": "simple",
```

Comment:
`"simple"` is the default style for path-parameters so there's no need to set it explicitly.